docs/spec.md

```diff
@@ -4963,8 +4977,12 @@ More formally:
 * (C9) `select` has type `(tensor<E>, tensor<E>) -> tensor<i1>` where
        `E = element_type(operand)`.
 * (C10) `scatter` has type `(tensor<E>, tensor<E>) -> tensor<E>` where
-        `E = element_type(operand)`.
-* (C11) `type(operand) = type(result)`.
+  `is_integer(element_type(operand)) = is_integer(element_type(E))` or
```

Comment:
Not sure if you want to define a name for this constraint and share it between 3 occurrences.

Reply:
I have a function on Type with name `same_elementtype_ignoring_bitwidth` for reusing the common. The name, however, is not representative of  how we compare the quantized element types, where we just checks for the equality of expressed_type and allowing the bitwidth, quantized dimension and quantization parameter to differ.

Other alternatives like   `same_elementtype_ignoring_bitwidth_quantized_dimension_and_parameter` is quite mouthful. I am looking for a better alternative and get back on that.